### ISSUE TITLE
Configure the component source to be local directory contents

### ISSUE THREAD

**Issue by jorgemoralespou** [redhat-developer/ocdev#221]
When I create a component I want to be able to specify that the source will the local directory, so when I push, the local directory will be pushed.

```
$ od create wildfly --local
$ od push
# Push local directory
```
Subtask of #169

**Comment by kadel** [comment]
Currently this is implemented a little bit differently.
There is an `--dir` option that controlls if it should be local or not.
Default value  for `--dir` value is ` ./`

after #224 is merged it will work like this

```
$ od create wildfly
```
Creates and pushes current dir.  It is equivalet to running `od create wildfly --dir ./`

```
$ od push
```
Pushes current dir to currently active component. There is also optional `--dir` option to allows to specify other directory than current.

@jorgemoralespou Should we change it to use `--local`?  But if `--local` is just a binary flag, than we will lose abitity to specify other directory than current one.

**Comment by jorgemoralespou** [comment]
@kadel you would never want to push to a component if you're not in the working directory. That was the idea we had in mind.

You code, you push, you code, you push.

If you have an arbitrary directory that you can set, then you might get some unexpected behavior in some cases, IMHO.

cc/ @marekjelen @gshipley what do you guys think?

**Comment by marekjelen** [comment]
Setting a dir as option seems OK to me, if you would be doing some kind of automation, then it might be more convenient to specify path, rather then `cd` somewhere; however, the default flow should not require me to specify it.

**Comment by kadel** [comment]
closed in #247 and #246
